coordinator/schedulepod/redis/service.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
```

Comment:
why not just use the quickstart helm chart?

Reply:
i did this back when I was having issues with getting quickstart working on my machine, so i made something lighter weight to test quickly, but yes it makes sense to do it against the existing quickstart helm chart